boards/esp32-heltec-lora32-v2/Makefile.dep

```diff
@@ -0,0 +1 @@
+include $(RIOTBOARD)/common/esp32/Makefile.dep
```

Comment:
Here you could also add (maybe it has other implication with esp stuff ?):
```mk
ifneq (,$(filter netdev_default,$(USEMODULE)))
  USEMODULE += sx1276
endif
```

Reply:
Hm, ..., not sure what the most usual case is. Default network interface might be also `esp_now` or `esp_wifi`.

Reply:
Why is it either or?
With this you'd get both `sx1276` *and* `esp_now`.

But this can be added when #12994 is merged so the user doesn't have to be careful not to forget to adding `GNRC_NETIF_NUMOF=2`, or else run into an assert failure when the pre-allocated netif slots run out.